### PR TITLE
[1LP][RFR] Adding service dialog manual test

### PR DESCRIPTION
**Pull request by niyazRedhat**
Purpose or Intent
=================

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit d65e2e736f3266955ffd88bfaa38b08c5c0a2509:

`cfme/tests/services/test_dialog_element_in_catalog.py`:
- :red_circle: Line 193:5: **N802** *function name should be lowercase*
- :red_circle: Line 588:5: **N802** *function name should be lowercase*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 1]
```diff
@@ -115,3 +115,608 @@ def test_validate_not_required_dialog_element():
...
+        expectedResults:
+            1.
+            2.
+            3. Populated "text area" fields should pass correct value  to ansible tower templates
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_dialog_editor_allows_you_to_modify_field_and_exit_without_saving_dialog():
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Possible to shorten this test case name? How about just `test_dialog_editor`? I don't think the test name needs to specify everything that the test_case does, that's what the testSteps are for.

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 2]
```diff
@@ -115,3 +115,608 @@ def test_validate_not_required_dialog_element():
...
+            1.
+            2.
+            3.
+            4. The UI should confirm that you want to exit without saving your dialog
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_set_specific_dates_and_time_in_timepicker_in_service_dialog():
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: How about just `test_service_dialog_timepicker`?

Comment (niyazRedhat) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: @john-dupuy I have already started automating test related to dialog and services which I didn't include here in this PR and these test will be my next target.

also made changes Please have a look

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 3]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/16h
+        testSteps:
+            1. Import Datastore and dialog
+            2. Add service catalog with above created dialog
+            3. In Order page of service
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Should this step be `Navigate to order page of service`?

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 4]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Import Datastore and dialog
+            2. Add service catalog with above created dialog
+            3. In Order page of service
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Same question as above

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 5]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+            2.
+            3. dynamic field shouldn't be blank
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1705021])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_refresh_of_dynamic_field_contains_word_password():
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Can this test case be merged into the previous test case? They seem to be testing the same page.

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 6]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Go to Services > Catalogs > Catalog Items accordion
+            2. Configuration > Add a New Catalog Item, choose some Catalog Item type
+            3. Fill in the required info and click on Add button
+            4. After successful saving the Catalog Item, click on the same Catalog Item in list
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Small typo `successfully`

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 7]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+        initialEstimate: 1/6h
+        testSteps:
+            1. create an aws template with an optional value "timeout"
+            2. create a dialog that will offer an option to overwrite "timeout"
+            with a custom value typed at input
+            3. provision using a non-zero value in timeout
+        expectedResults:
+            1.
+            2.
+            3. the value inputted should be passed
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: `input` is weird and its past tense is just `input`

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 8]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+    Bugzilla:
+        1684092
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Added ec2 provider
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: I would use present tense for `testSteps`, but I'll leave that as optional.

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 9]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Import DataStore and Dynamic Dialog
+            2. Add catalog item with above dialog
+            3. Order the service
+        expectedResults:
+            1.
+            2.
+            3. Dynamic dialog all functionality should work
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: _Optional_: `All dynamic dialog functionality should work`

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 10]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+    """
+    pass
+
+
+@pytest.mark.meta(coverage=[1706600])
+@pytest.mark.manual
+@pytest.mark.tier(2)
+def test_dynamic_dialogs_on_service_request():
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Can these tests with the same `testSteps` be collated into a single test case? Also, it is unclear just from the testSteps what the test is testing, though I suppose the BZ will give some more information.

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 11]
```diff
@@ -115,3 +116,658 @@ def test_validate_not_required_dialog_element():
...
+        assignee: nansari
+        startsin: 5.11
+        casecomponent: Services
+        initialEstimate: 1/16h
+        testSteps:
+            1. create a service dialog
+            2. In service dialog element option page
+        expectedResults:
+            1.
+            2. Load_values_on_init button should always be enable
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: small typo `enabled`

File: cfme/tests/services/test_dialog_element_in_catalog.py (hunk 5 above)
Comment (niyazRedhat) [reply cfme/tests/services/test_dialog_element_in_catalog.py]: Here in service dialog lots of tests testing on the same page but it has different different scenario and automate method so we can't merge it.

**Comment by dajoRH** [comment]
Lint report for commit cc5a4488e79491eca883c29a76491cfb7e2b5a92:

`cfme/tests/services/test_dialog_element_in_catalog.py`:
- :heavy_exclamation_mark: Line 793:46: **W291** *trailing whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/services/test_dialog_element_in_catalog.py [hunk 12]
```diff
@@ -115,3 +116,686 @@ def test_validate_not_required_dialog_element():
...
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Import DataStore and Dynamic Dialog
+            2. Add catalog item with above dialog
+            3. Navigate to order page of service
+            4. In service Order page
+            3. Order the service
```
Comment (john-dupuy) [inline cfme/tests/services/test_dialog_element_in_catalog.py]: Small typo, these steps seem mixed up.

**Pull request merged**